tutorials/asr/05_Online_Noise_Augmentation.ipynb

```diff
@@ -0,0 +1,1530 @@
+        "def load_audio() -> segment.AudioSegment:\n",
+        "    filename = 'Kevin_MacLeod_-_Vibe_Ace.wav'\n",
+        "    filepath = os.path.join('media', filename)\n",
+        "    sample_segment = segment.AudioSegment.from_file(filepath, target_sr=sr)\n",
+        "    return sample_segment\n",
+        "\n",
+        "sample_segment = load_audio()\n",
+        "ipd.Audio(sample_segment.samples, rate=sr)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "hTnf1g1y63wZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "## White Noise Perturbation\n",
+        "\n",
+        "White Noise perturbation is performed by the following steps : <br>\n",
+        "1) Randomly sample the amplitude of the noise from a uniformly distributed range (defined in dB) <br>\n",
+        "2) Sample gaussian noise (mean = 0, std = 1) with same length as audio signal <br>\n",
+        "3) Scale this gaussian noise by the amplitude (in dB scale) <br>\n",
+        "4) Add this noise vector to the original sample\n",
+        "\n",
+        "Notably, the original signal should not have a \"hissing sound\" constantly present in the perturbed version."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "2jaPQyUY65ij",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "white_noise = perturb.WhiteNoisePerturbation(min_level=-50, max_level=-30)\n",
+        "\n",
+        "# Perturb the audio file\n",
+        "sample_segment = load_audio()\n",
+        "white_noise.perturb(sample_segment)\n",
+        "\n",
+        "ipd.Audio(sample_segment.samples, rate=sr)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b8KVSN-l69ju",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Gain Perturbation\n",
+        "\n",
+        "Gain perturbation is performed by the following steps : <br>\n",
+        "1) Randomly sample the gain factor of the signal from a uniformly distributed range (defined in dB) <br>\n",
+        "2) Scale this original signal by the gain factor (in dB scale) <br>\n",
+        "\n",
+        "Notably, the tone of the original audio should sound slightly different as compared to the gain perturbed sample."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "rMSA1Y8o696j",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "gain = perturb.GainPerturbation(min_gain_dbfs=25, max_gain_dbfs=50)\n",
+        "\n",
+        "# Perturb the audio file \n",
+        "sample_segment = load_audio()\n",
+        "gain.perturb(sample_segment)\n",
+        "\n",
+        "ipd.Audio(sample_segment.samples, rate=sr)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
```

Comment:
Could you also delete this section about Gain perturbation for now.

Reply:
Done